apps/modernization-ui/src/components/DemographicsForm/Ethnicity.tsx

```diff
@@ -13,6 +14,13 @@ export const EthnicityForm = ({ setEthnicityForm }: any) => {
         setEthnicityForm();
     };
 
+    const [selectedEthinicity, setSelectedEthinicity] = useState<'2135-2' | '2186-5' | 'UNK' | 'NOT-TO-ANS' | ''>('');
```

Comment:
I'm not sure we can set the type like this as different NBS installations could have different / additional ethnicity options.

Reply:
Agreed, I think this is good for now because this can ultimately be handled by the backend returning a property indicating that detailed ethnicities exist.